dotnet/src/support/Events/EventFiringWebDriver.cs

```diff
@@ -842,15 +843,24 @@ public EventFiringNavigation(EventFiringWebDriver driver)
             }
 
             /// <summary>
-            /// Move the browser back
+            /// Move the browser back a
             /// </summary>
             public void Back()
+            {
+                Task.Run(this.BackAsync).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Use `AsyncHelper`

Reply:
Ah, couldn't access it from support class because it's internal. I guess I'll make it public.

Reply:
I would say it is better to keep the same own copy of AsyncHelper in Support assembly. It is not good idea to make AsynHelper public.

Reply:
I started with copying it, but other classes in the internal directory are public and follow this pattern. Not ideal, but neither is the same code in multiple places.